sematic/plugins/storage/local_storage.py

```diff
@@ -21,6 +27,10 @@
 _PLUGIN_VERSION = (0, 1, 0)
 
 
+class LocalStorageSettingsVar(AbstractPluginSettingsVar):
+    LOCAL_STORAGE_PATH = "LOCAL_STORAGE_PATH"
```

Comment:
Please update [the public-facing documentation](https://docs.sematic.dev/diving-deeper/local-execution#artifact-storage) with this option.